tsconfig.json

```diff
@@ -16,12 +16,13 @@
     "noImplicitReturns": true,
     "resolveJsonModule": true,
     "esModuleInterop": true,
+    "moduleResolution": "node",
     "composite": true,
     "incremental": true,
     "typeRoots": [
       "./node_modules/@types",
       "./custom_typing"
     ]
   },
-  "includes": ["**/*.ts", "**/*.tsx"]
+  "include": ["**/*.ts", "**/*.tsx", "src/**/*.json"]
```

Comment:
does adding the json fuck with the linter rules?

Reply:
Kind of, it's just to satisfy some TypeScript import rules regarding JSON